integration_tests/k8s/k8sapi.go

```diff
@@ -0,0 +1,140 @@
+package k8s
```

Comment:
we should bind all the function with a struct thatt contains `*kubernetes.Clientset`.
Better to avoid functions please use methods

Reply:
If we are creating a wrapper over k8s client exposed functions need to think about packaging.

Reply:
These are the core functionality instead of putting it here we can add it inside `pkg` so that other can use it.
If you put it here it is accessible in other packages but import path will not be meaningful.

Reply:
@imazik , currently we will keep the BDD code and NDM business logic code separate. If they converge on the same methods, then we will move into pkg